kolibri/deployment/default/settings/dev.py

```diff
@@ -11,3 +11,10 @@
 ROOT_URLCONF = 'kolibri.deployment.default.dev_urls'
 
 DEVELOPER_MODE = True
+
+# Create a dummy cache for each cache
```

Comment:
thus, nulls the caches, right?

Reply:
Yes, I am open to arguments that we should enforce caching in development to expose caching induced bugs to devs, but this will prevent having to do superfluous saves on python files to restart the devserver just to clear the cache.

Reply:
nah, I just wanted to confirm I had understood your purpose correctly.